### ISSUE TITLE
r1cs-bulletproofs-presentation-fixes

### ISSUE THREAD

**Issue by hansieodendaal** [tari-labs/tari-university#251]
Hi @Empiech007,

Some comments on the [presentation](https://gitpitch.com/tari-labs/tari-university/master?p=/src/cryptography/r1cs-bulletproofs#/):

1. Use CSS from `assets\tari-b.css` to enhance the layout and presentation of the slides.

2. Also use local CSS to enhance the layout and presentation of the slides, e.g. add to the beginning of the `.md` file:
```
<head>
<style>
div.LineHeight20per {
  line-height: 20%;
}
div.LineHeight100per {
  line-height: 100%;
}
div.LineHeight200per {
  line-height: 200%;
}
div.tab {
  "font-size:0.8em;"
}
</style>
</head>
```

3. Slide 11: Last bullet should not be a bullet, i.e. use `@div[text-left]` from `assets\tari-b.css`

4. Slide 13: Rendering error in formula in the 2nd bullet

5. Slide 14: 
   - Replace/remove additional full stop `.` in sentence.
   - Spacing required between 1st par. and table heading, i.e use `<div class="LineHeight100per"> <br></div>` to insert a break.
   - Table too big to fit the page - use a smaller font, i,e, use `<div class="tab">MyCellText</div>` for every cell entry.

6. Slide 16: 
    - Remove `####` in front of image URL
    - Resize image, i.e. use `@div[s650px]` or other from `assets\tari-b.css`  to properly fit page

7. Other slides with images after the previous one:
    - ditto Slide 16
    - ditto Slide 16

8. Slide 29: Too much text on one page

**Comment by Empiech007** [comment]
Fixed most things. 
Left with two: 
- the equation in Slide 13, and 
- the table in slide 14 is still big despite the fixes.

**Comment by hansieodendaal** [comment]
@Empiech007, please create a PR for this, then we can comment further there. When we approve that PR, we can mention this issue and it will be closed as well. Thanks.

**Comment by hansieodendaal** [comment]
Closed with #252
